README.md

```diff
@@ -1,70 +1,65 @@
-# Getting Started with Create React App
+# React and Redux
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
 
-## Available Scripts
+## Set-up GitHub Actions
 
-In the project directory, you can run:
+This GitHub Action is going to run [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/) to help you find style issues.
 
-### `npm start`
+[Stylelint](https://stylelint.io/) is a linter for your stylesheets that helps you avoid errors and enforce conventions.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[ESLint](https://eslint.org/) is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+Please do the following **steps in this order**:
 
-### `npm test`
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
```

Comment:
- [ ] Kindly make sure to follow the microverse readme template and correct headers.